src/test/scala/com/microsoft/hyperspace/index/IndexManagerTests.scala

```diff
@@ -314,7 +314,9 @@ class IndexManagerTests extends HyperspaceSuite with SQLHelper {
       hyperspace.optimizeIndex(indexConfig.indexName)
 
       // Check if latest log file is updated with newly created index files.
-      validateMetadata("index", Set("v__=2"))
+      // Since we appended 3 records for incremental refresh (v_==1) and optimized as v_==2,
+      // some of index data files should be from v__=0.
+      validateMetadata("index", Set("v__=0", "v__=2"))
```

Comment:
Is this an existing test bug?

Reply:
No, the current version of optimizeIndex rewrites single files also. This PR fixes it.